ndn_hydra/repo/utils/garbage_collector.py

```diff
@@ -0,0 +1,57 @@
+import time
+import logging
+from ndn.svs import SVSync
+from ndn_hydra.repo.modules.global_view import GlobalView
+from ndn_hydra.repo.group_messages.remove import RemoveMessageTlv
+from ndn_hydra.repo.group_messages.message import Message, MessageTypes
+
+
+def collect_db_garbage(global_view: GlobalView, svs: SVSync, config: dict, logger: logging.Logger) -> None:
+    """
+    Removes files that have not been accessed in the last month from a node's databases.
+    """
+    logger.info("GARBAGE COLLECTOR: Collecting DB garbage...")
+
+    ONE_MONTH = 60*60*24*30 # seconds in one month
```

Comment:
We want Hydra to be expansive, ONE_MONTH might be too little, too long, etc.

Lets move this in the current config (the default config in main/main.py) and call it "file_expiration" and let it be in terms of hours (0 meaning forever).